packages/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -91,22 +95,6 @@ export class TriggerEvent {
       triggerIdentifier: command.identifier,
     });
 
-    /*
-     * Makes no sense to execute anything if template doesn't exist
-     * TODO: Send a 404?
-     */
-    if (!template) {
-      const message = 'Notification template could not be found';
-      const error = new ApiException(message);
-      Logger.error(error, message, LOG_CONTEXT);
-      throw error;
-    }
```

Comment:
this part was redundant because we check it in ParseEventRequest use case and throw an exception